xv83.md

```diff
@@ -95,8 +103,6 @@ is included in the README?
 
```

Comment:
Yes, I think a README with most of that information would definitely be required. In particular it's really important with this type of data that caveats and limitations be very clearly indicated - for example it's likely that "authoritative" data is (probably?) not appropriate for publishing on, as it is "pre-QC" - data on which publications are based should be that in `pd00` or elsewhere published formally.